apps/hpc-ftsadmin/src/app/components/flow-form.tsx

```diff
@@ -0,0 +1,4094 @@
+  parentFlow: forms.InputEntryType | null;
+  childFlow: forms.InputEntryType[];
+}
+
+type UploadedItem = fileUpload.FileUploadResult | FileAssetEntityType;
+
+const reportChannelSchema = t.type({
+  value: t.intersection([codecs.NON_EMPTY_STRING, t.string]),
+  displayLabel: t.intersection([codecs.NON_EMPTY_STRING, t.string]),
+});
+
+const reportDetailsSchema = t.type({
+  reportedOrganization: INPUT_SELECT_VALUE_TYPE,
+  reportedDate: t.intersection([codecs.NON_EMPTY_STRING, t.string]),
+  reportChannel: t.intersection([codecs.NON_EMPTY_STRING, reportChannelSchema]),
+  reportFileTitle: t.intersection([codecs.NON_EMPTY_STRING, t.string]),
+});
+
+const validationSchema = t.type({
+  amountUSD: t.number,
+  flowStatus: INPUT_SELECT_VALUE_TYPE,
+  flowDescription: t.intersection([codecs.NON_EMPTY_STRING, t.string]),
+  firstReported: t.string,
+  flowDate: t.intersection([codecs.NON_EMPTY_STRING, t.string]),
+  sourceOrganizations: INPUT_SELECT_VALUE_TYPE,
+  sourceUsageYears: INPUT_SELECT_VALUE_TYPE,
+  destinationOrganizations: INPUT_SELECT_VALUE_TYPE,
+  destinationUsageYears: INPUT_SELECT_VALUE_TYPE,
+  reportDetails: reportDetailsSchema,
+});
+
+interface Props {
+  currentVersionData: flowsResponse.FlowREST | null;
+  environment: Environment;
+  isEdit: boolean;
+  initialValue: FormValues;
+  prevDetails?: ReportDetailType[];
+  versionData?: VersionDataType[];
+  flowId?: string;
+  versionId?: string;
+  isRestricted: boolean;
+  errorCorrection?: boolean;
+  inputEntries: InputEntriesType;
+  initializeInputEntries: () => void;
+  rejectInputEntry: (key: string) => void;
+  currentVersionID?: number;
+  currentFlowID?: number;
+  currentVersionActiveStatus?: boolean;
+  isPending?: boolean;
+  isSuperseded?: boolean;
+  isCancelled?: boolean;
+  isCancellation?: boolean;
+  isNewPending?: boolean;
+  isUpdatePending?: boolean;
+  canReactive?: boolean;
+  isErrorCorrection?: boolean | null;
+  isApprovedFlowVersion?: boolean | null;
+  pendingFieldsallApplied?: boolean;
+  allFieldsReviewed?: boolean;
+  pendingVersionV1?: boolean;
+}
+
+const StyledRepOrgLink = tw.div`
+  flex
+  ml-1
+`;
+
+const StyledAddChildWarning = tw.div`
+  border border-solid border-yellow-600
+  flex
+  rounded-md
+  text-gray-900
+  bg-yellow-100 bg-opacity-50
+`;
+
+const StyledAddChildWarningIconDiv = tw.div`
+  flex
+  items-center
+  p-1
+`;
+const StyledAddChildWarningText = tw.span`
+  py-3 px-3 pb-3
+`;
+const StyledLayoutRow = tw.div`
+flex
+`;
+const StyledHalfSection = tw.div`
+w-1/2
+`;
+const StyledFullSection = tw.div`
+w-full
+mb-6
+`;
+const StyledRow = tw.div`
+flex
+gap-4
+w-full
+`;
+const StyledFormRow = tw.div`
+flex
+gap-2
+w-full
+items-center
+`;
+const StyledAnchor = tw.a`
+underline
+ml-[15px]
+opacity-100
+`;
+const StyledAnchorDiv = tw.div`
+text-right
+w-full
+`;
+const StyledRadioDiv = tw.div`
+relative
+w-full
+`;
+const StyledFieldset = tw.fieldset`
+w-full
+box-border
+rounded-xl
+mt-[16px]
```

Comment:
(There is an extension for VSCode called 'Tailwind CSS IntelliSense' it could help you with suggestions, it would tell you that `mt-4` equals to 16px)
```suggestion
mt-4
```